tests/unit/stream_alert_rule_processor/test_config.py

```diff
@@ -112,3 +119,28 @@ def test_config_valid_types():
     config = load_config()
 
     _validate_config(config)
+
+def test_load_threat_intel_conf():
+    """Config - valid config for Threat Intel"""
+    enable_threat_intel, mapping = load_threat_intel_conf()
+    expected_mapping = {
+        'destinationAddress': 'ip',
+        'destinationDomain': 'domain',
+        'deviceAddress': 'ip',
+        'fileHash': 'md5',
+        'sourceAddress': 'ip'
+    }
+    assert_equal(enable_threat_intel, False)
+    assert_equal(mapping, expected_mapping)
+
+@raises(ConfigError)
+def test_load_threat_intel_conf_nonexist():
+    """Config Error - Threat Intel configuration file is not existing"""
```

Comment:
`Threat Intel configuration file is not existing` -> `Threat Intel configuration file does not exist`